package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mobify-platform-sdks",
-  "version": "1.5.0-dev",
+  "version": "1.5.0",
   "lockfileVersion": 1,
```

Comment:
This is still 1.5.0. It should be 1.6.0-dev

Reply:
Good catch! The version was fixed after I ran the bump-version script for the second time.